src/vm/gccover.cpp

```diff
@@ -73,6 +73,97 @@ static MethodDesc* getTargetMethodDesc(PCODE target)
 }
 
 
+bool IsGcCoveregeInterruptInstruction(LPVOID instrPtr)
+{
+#if defined(_TARGET_ARM64_)
+    UINT32 instrVal = *reinterpret_cast<UINT32*>(instrPtr);
+    switch (instrVal)
+    {
+    case INTERRUPT_INSTR:
+    case INTERRUPT_INSTR_CALL:
+    case INTERRUPT_INSTR_PROTECT_RET:
+        return true;
+    default:
+        return false;
+    }
+#elif defined(_TARGET_ARM_)
+    
+    SLOT instSlot = reinterpret_cast<SLOT>(instrPtr);
+    size_t instrLen = GetARMInstructionLength(instSlot);
```

Comment:
As I understand now `SLOT == PBYTE`, because we do not have ia64.
There is a mix of SLOT, PBYTE, TADDR, and PCODE in this file. I thought that the SLOT is more common and used it in `IsGcCoverageInterruptInstruction`, `IsOriginalInstruction` etc. Do we want to use PBYTE instead?

Reply:
Also, we have a mix of WORD, UINT16 as well. I think this file is under our team control, do we have a preference?

Reply:
I'm not as familiar with the conventions in these files. However, `SLOT` doesn't make much sense to me. At least, when I naively read `SLOT`, I don't think `pointer to instruction`.

Reply:
I think it is the VM convention.
For example:
```
gccover.cpp:
static SLOT getTargetOfCall(SLOT instrPtr, PCONTEXT regs, SLOT*nextInstr);
bool isCallToStopForGCJitHelper(SLOT instrPtr);
excep.cpp:
HRESULT SetIPFromSrcToDst(Thread *pThread, SLOT addrStart,       // base address of method
```
but I can change it to explicit PBYTE because there is no difference between them:
```
// This will need ifdefs for non-x86 processors (ia64 is pointer to 128bit instructions)!
#define SLOT    PBYTE
```

Reply:
Change all `SLOT` in this file to `PBYTE`.